source/Generic/SplashScreen/SplashScreen.cs

```diff
@@ -257,6 +283,7 @@ private void CreateSplashVideoWindow(bool showSplashImage, string videoPath, str
                 currentSplashWindow.Content = new SplashScreenImage { DataContext = new SplashScreenImageViewModel(generalSplashSettings, splashImagePath, logoPath) };
                 PlayniteApi.Dialogs.ActivateGlobalProgress((a) =>
                 {
+                    ActivateSplashWindow(currentSplashWindow);
```

Comment:
Just wondering, why is this method being called twice? Also why aren't the `Activate` and `CaptureMouse` methods being called (or replacing) where the `Show` method is being used above?

Reply:
`ActivateGlobalProgress` activates progress window. we must activate splash window **after** progress window is shown. it seems that we can do it only inside anonymous function that is passed to `ActivateGlobalProgress` call.
and `ActivateGlobalProgress` is called **twice**, for video and then for splash after video.